.github/workflows/buildandtest.yml

```diff
@@ -0,0 +1,137 @@
+name: Build & Test
+on:
+  push:
+    branches:
+      - master
+      - release/**
+  pull_request:
+
+jobs:
+  unit-tests:
+    name: Unit Tests ${{matrix.name}}
+    runs-on: macos-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        
+        include:
+          # SentrySystemEventsBreadcrumbsTest only run on Catalyst and iOS
+          # so the coverage report shows a reduced coverage.
+          - name: macOS
+            destination: platform=macOS
+            pushCodecov: false
+          - name: Mac Catalyst
+            destination: platform=macOS,variant=Mac Catalyst  
+            pushCodecov: true
+          - name: iOS  
+            destination: platform=iOS Simulator,OS=latest,name=iPhone 11 Pro
+            pushCodecov: true
+          - name: tvOS  
+            destination: platform=tvOS Simulator,OS=14.0,name=Apple TV 4K
+            pushCodecov: false    
+          # Can't run tests on watchOS because XCTest is not available  
+
+    steps:
+      - uses: actions/checkout@v2
+      - run: ./scripts/ci-select-xcode.sh
+      - name: Running tests
+        run: >-
+          xcodebuild 
+          -workspace Sentry.xcworkspace 
+          -scheme Sentry 
+          -configuration Debug 
```

Comment:
Is it possible to test in release more? That's the type that gets shipped so always good to run CI tests on that if the platform supports it.